themes/default/content/blog/converting-bicep-to-pulumi/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Converting Bicep code to Pulumi"
+date: 2023-09-27
+draft: false
+meta_desc: In this article, we will look at a new language converter that takes Bicep code and converts it to any of supported Pulumi languages. 
+meta_image: meta.png
+authors: ["zaid-ajaj"]
+tags: ["community", "bicep", "pulumi", "convert"]
+---
+
+Bicep is DSL developed by Microsoft to simplify the authoring of ARM templates and deploy resources to Azure. Today I will be sharing with you a new Pulumi converter plugin that I have been working on that converts Bicep code to any of the supported Pulumi languages.
```

Comment:
```suggestion
Bicep is a DSL developed by Microsoft to simplify the authoring of ARM templates and deploy resources to Azure. Today I will be sharing with you a new Pulumi converter plugin that I have been working on that converts Bicep code to any of the supported Pulumi languages.
```

Reply:
Thanks a lot for the finding the suggestions 🙏 I've applied those to the blog post accordingly ✅